azure-devops/azext_devops/dev/team/service_endpoint.py

```diff
@@ -172,3 +199,27 @@ def create_service_endpoint(service_endpoint_configuration,
     import json
     service_endpoint_to_create = json.loads(in_file_content)
     return client.create_service_endpoint(service_endpoint_to_create, project)
+
+
+def update_service_endpoint(id, enable_for_all=None, organization=None,  # pylint: disable=redefined-builtin
+                            project=None, detect=None):
+    """Update a service endpoint
+    :param id: ID of the service endpoint.
+    :type id: str
+    """
+    if enable_for_all is None:
+        raise CLIError('Atleast one property to be updated must be specified.')
```

Comment:
'enable_for_all' is mandatory ? [Since its the only argument]

Reply:
Future proof... we will end up adding new params to the command and all should remain optional. 

There is error handling right now if it is not set.